logrusr.go

```diff
@@ -123,6 +131,7 @@ func listToLogrusFields(keysAndValues ...interface{}) logrus.Fields {
 		k, v := keysAndValues[i], keysAndValues[i+1]
 
 		if s, ok := k.(string); ok {
+
 			// Try to avoid marshaling known types.
```

Comment:
Remove unnecessary empty line.

```suggestion
		if s, ok := k.(string); ok {
			// Try to avoid marshaling known types.
```